ci/build_python.sh

```diff
@@ -12,12 +12,18 @@ rapids-print-env
 rapids-logger "Begin py build"
 
 CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+PY_VER=${RAPIDS_PY_VERSION//./}
+LIBCUDF_CHANNEL=$(rapids-get-artifact ci/cudf/pull-request/12587/046025a/cudf_conda_cpp_cuda11_$(arch).tar.gz)
+CUDF_CHANNEL=$(rapids-get-artifact ci/cudf/pull-request/12587/046025a/cudf_conda_python_cuda11_${PY_VER}_$(arch).tar.gz)
```

Comment:
Same question as above. Python `cuml` does use `cudf` so it makes more sense, but these seem like temporary changes for testing, right?

Reply:
Yup, same as above